packages/bappo-components/src/components/Background/Background.tsx

```diff
@@ -1,21 +1,20 @@
-// @flow
-
 import * as React from 'react';
 
 import { styled } from '../../apis/Style';
 import View from '../../primitives/View';
 
 type Props = {
-  children?: string,
-};
-
-const Background = ({ children }: Props) => {
-  return <StyledView selectable>{children}</StyledView>;
+  children?: string;
+  testID?: string;
 };
 
-Background.defaultProps = {};
-
-export default Background;
+export default function Background({ children, testID }: Props = {}) {
```

Comment:
{ children, testID }: Props = {}   Hi Emil, should we keep the {} as default value? I am not sure about that, could went me through that when you free? Thanks

Reply:
Thanks, I've also removed the `{}` as default value.